stripe/http_client.py

```diff
@@ -494,24 +516,35 @@ def close(self):
         pass
 
 
+class _Proxy(TypedDict):
+    http: Optional[ParseResult]
+    https: Optional[ParseResult]
+
+
 class PycurlClient(HTTPClient):
     name = "pycurl"
+    _proxy: _Proxy
```

Comment:
Should we have a separate field for parsed proxy instead of mutating its type?

Reply:
Yes!

Reply:
Fixed with [_proxy -> _parsed_proxy](https://github.com/stripe/stripe-python/pull/1020/commits/ee66bb0a8721004455ab4e3f27571af9aba3d1fb)